src/inline-help.scss

```diff
@@ -44,186 +33,27 @@ $block: #{$fd-namespace}-inline-help;
     position: absolute;
     left: 0;
     border-radius: 50%;
-
-    @include fd-var-color("color", fd-color("text", 5), --fd-inline-help-icon-color);
-    @include fd-var-color("background-color", $fd-tooltip-icon-background-color, --fd-inline-help-icon-background-color);
-
+    color: var(--sapContent_ContrastIconColor);
+    background-color: $fd-tooltip-icon-background-color;
     text-align: center;
   }
 
   // ELEMENTS *******************************************
   &__content {
     @include fd-type("-1");
```

Comment:
`fd-type` is no longer supported. These were the values before. You can replace them by `font-size: var(--sapFontSmallSize)` and `line-heigh: normal`
![Screen Shot 2020-05-28 at 4 11 40 PM](https://user-images.githubusercontent.com/39598672/83188577-eeb17a00-a0fd-11ea-8168-0461e0a6393a.png)